src/components/Form/DatePicker/DatePicker.stories.tsx

```diff
@@ -0,0 +1,275 @@
+        incrementIcon: { control: 'text' },
+        inline: { control: 'boolean' },
+        inputClassName: { control: 'text' },
+        inputId: { control: 'text' },
+        inputRef: { control: 'text' },
+        inputStyle: { control: 'object' },
+        keepInvalid: { control: 'boolean' },
+        locale: { control: 'text' },
+        mask: { control: 'text' },
+        maskSlotChar: { control: 'text' },
+        maxDate: { control: 'date' },
+        maxDateCount: { control: 'number' },
+        minDate: { control: 'date' },
+        monthNavigator: { control: 'boolean' },
+        name: { control: 'text' },
+        nextIcon: { control: 'text' },
+        numberOfMonths: { control: 'number' },
+        panelClassName: { control: 'text' },
+        panelStyle: { control: 'object' },
+        placeholder: { control: 'text' },
+        prevIcon: { control: 'text' },
+        readOnlyInput: { control: 'boolean' },
+        required: { control: 'boolean' },
+        selectOtherMonths: { control: 'boolean' },
+        shortYearCutoff: { control: 'text' },
+        showButtonBar: { control: 'boolean' },
+        showIcon: { control: 'boolean' },
+        showMillisec: { control: 'boolean' },
+        showMinMaxRange: { control: 'boolean' },
+        showOnFocus: { control: 'boolean' },
+        showOtherMonths: { control: 'boolean' },
+        showSeconds: { control: 'boolean' },
+        showTime: { control: 'boolean' },
+        showWeek: { control: 'boolean' },
+        stepHour: { control: 'number' },
+        stepMillisec: { control: 'number' },
+        stepMinute: { control: 'number' },
+        stepSecond: { control: 'number' },
+        style: { control: 'object' },
+        tabIndex: { control: 'number' },
+        timeOnly: { control: 'boolean' },
+        todayButtonClassName: { control: 'text' },
+        tooltip: { control: 'text' },
+        tooltipOptions: { control: 'object' },
+        touchUI: { control: 'boolean' },
+        transitionOptions: { control: 'object' },
+        view: { control: { type: 'select', options: ['date', 'month', 'year'] } },
+        viewDate: { control: 'date' },
+        visible: { control: 'boolean' },
+        yearNavigator: { control: 'boolean' },
+        yearRange: { control: 'text' },
+        formatDateTime: { control: 'text' },
+        parseDateTime: { control: 'text' },
+        dateTemplate: { control: 'text' },
+        decadeTemplate: { control: 'text' },
+        footerTemplate: { control: 'text' },
+        headerTemplate: { control: 'text' },
+        monthNavigatorTemplate: { control: 'text' },
+        yearNavigatorTemplate: { control: 'text' },
+    },
+} as Meta;
+
+const Template: StoryFn<DatePickerProps> = (args) => <DateField {...args} />;
+
+/**
+ * Default story: Renders the DateField with default props.
+ */
+export const Default = Template.bind({});
+Default.args = {
+    label: 'Select Date',
```

Comment:
The **name** is a required prop but you didn't use it?
![Screenshot 2024-06-20 150811](https://github.com/gandhiFinforz/FN-UI-Framework/assets/173181168/71e3afe4-6ef6-42b4-a0e0-48631ffb441a)